custom/icds_reports/ucr/data_sources/dashboard/remove_member_form.json

```diff
@@ -0,0 +1,103 @@
+{
+  "domains": [
+    "icds-dashboard-qa",
+    "sankalp_cas",
+    "zohaib-sandbox",
+    "akshita-sandbox",
+    "sunaina-sandbox",
+    "laurence-project-1",
+    "jessica-icds-cas",
+    "marissa-test",
+    "derek-icds-sandbox",
+    "priyanka-app",
+    "shrena-dev",
+    "aparatest",
+    "reach-sandbox",
+    "reach-dashboard-qa",
+    "reach-test",
+    "icds-cas",
+    "cas-lab"
+  ],
+  "server_environment": [
+    "india",
+    "icds"
+  ],
+  "config": {
+    "table_id": "static-remove_member",
+    "display_name": "Forms - Remove Member (REACH Dashboard)",
+    "referenced_doc_type": "XFormInstance",
+    "base_item_expression": {},
+    "configured_filter": {
+      "type": "boolean_expression",
+      "operator": "eq",
+      "expression": {
+        "type": "property_name",
+        "property_name": "xmlns"
+      },
+      "property_value": "http://openrosa.org/formdesigner/705d02eb71b1e3f493ecdb519f84b8c9cf9f5fbf"
+    },
+    "configured_indicators": [
```

Comment:
you will also need timeend column to filter the forms based on when they were submitted.

Reply:
could you also add state_id column.

Reply:
done